react-native.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  assets: ['./app/assets/fonts', 'react-native-vector-icons'],
+  assets: ['./app/assets/fonts'],
```

Comment:
Why you remove `react-native-vector-icons`?

Reply:
this will be included by autolinking, no need for manual linking